script/npm_lint_plugins.js

```diff
@@ -1,23 +1,91 @@
 #!/usr/bin/env node
 /* eslint-disable no-var */
+/* eslint-disable no-console */
 
 var fs = require('fs');
 
-var childProcess = require('child_process');
-var { packageJsonDirs } = require('./plugin_webpack_directories');
+const { spawnSync, execSync } = require('child_process');
+const path = require('path');
+
+var { packageJsonDirsObject } = require('./plugin_webpack_directories');
+
+var dirs = packageJsonDirsObject();
+var dirsKeys = [];
+var skippedDirs = [];
+var passedArgs = process.argv.slice(2);
+Object.keys(dirs).forEach(k => {
```

Comment:
Can we move it to `plugin_webpack_directories`? 
something like:
``` js
const skipGems = true;
var dirs = packageJsonDirsObject(skipGems);
```
or
``` js
var dirs = developmentPackageJsonDirs();
```
We will have a single place where we identify that the plugin is a gem. If we would change this logic later, it would be much easier.